contracts/GnosisSafe.sol

```diff
@@ -15,17 +15,17 @@ contract GnosisSafe is MasterCopy, BaseSafe, SignatureDecoder, SecuredTokenTrans
     using SafeMath for uint256;
 
     string public constant NAME = "Gnosis Safe";
-    string public constant VERSION = "0.1.0";
+    string public constant VERSION = "1.0.0";
 
     //keccak256(
     //    "EIP712Domain(address verifyingContract)"
     //);
     bytes32 public constant DOMAIN_SEPARATOR_TYPEHASH = 0x035aff83d86937d35b32e04f0ddc6ff469290eef2f1b692d8a815c89404d4749;
 
     //keccak256(
-    //    "SafeTx(address to,uint256 value,bytes data,uint8 operation,uint256 safeTxGas,uint256 dataGas,uint256 gasPrice,address gasToken,address refundReceiver,uint256 nonce)"
+    //    "SafeTx(address to,uint256 value,bytes data,uint8 operation,uint256 safeTxGas,uint256 baseGas,uint256 gasPrice,address gasToken,address refundReceiver,uint256 nonce)"
     //);
-    bytes32 public constant SAFE_TX_TYPEHASH = 0x14d461bc7412367e924637b363c7bf29b8f47e2f84869f4426e5633d8af47b20;
+    bytes32 public constant SAFE_TX_TYPEHASH = 0xbb8310d486368db6bd6f849402fdd73ad53d316b5a4b2644ad6efe0f941286d8;
```

Comment:
@Uxio0 this will require changes in the clients, so we need to add a way that the clients know which version to use for signing (maybe return the contract version with the estimates)

Reply:
Alternatively we could stick to dataGas (but imo this is a really bad name ... on the other side the user will not see it)

Reply:
I would prefer that clients query the GET `/safes/{address}/` endpoint, I don't like how the estimate endpoint is growing. Currently `masterCopy`, `nonce`, `threshold` and `owners` are returned, I will add `version` to that endpoint too. Would be ok for you @rmeissner?